### ISSUE TITLE
Application partition is too small

### ISSUE THREAD

**Issue by krgn** [nerves-project/nerves_system_x86_64#74]
### Environment

* Elixir version (`elixir -v`):  Elixir 1.9.1 (compiled with Erlang/OTP 22)
* Nerves environment: (`mix nerves.env --info`)
```
|nerves_bootstrap| Environment Package List

  Pkg:         custom_x86_64
  Vsn:         1.9.2
  Type:        system
  BuildRunner: {Nerves.Artifact.BuildRunners.Local, []}

  Pkg:         nerves_system_br
  Vsn:         1.9.5
  Type:        system_platform
  BuildRunner: {nil, []}

  Pkg:         nerves_toolchain_ctng
  Vsn:         1.6.0
  Type:        toolchain_platform
  BuildRunner: {nil, []}

  Pkg:         nerves_toolchain_x86_64_unknown_linux_musl
  Vsn:         1.2.0
  Type:        toolchain
  BuildRunner: {Nerves.Artifact.BuildRunners.Local, []}

|nerves_bootstrap| Loadpaths Start


Nerves environment
  MIX_TARGET:   custom_x86_64
  MIX_ENV:      dev

|nerves_bootstrap| Environment Variable List
  target:     custom_x86_64
  toolchain:  /home/k/.nerves/artifacts/nerves_toolchain_x86_64_unknown_linux_musl-linux_x86_64-1.2.0
  system:     /home/k/.nerves/artifacts/custom_x86_64-portable-1.9.2
  app:        /opt/build/sensor_event_daemon
```

* Additional information about your host, target hardware or environment that
  may help

Building the custom image in a ubuntu focal docker container for running in qemu using the shipped script.

### Current behavior

I'm building an artifact like this (after having added some packages):

`MIX_TARGET=custom_x86_64 mix nerves.artifact` 

In `fwup.conf` the application parition size is said to be 10GB, however when I run the image for my custom build the application partition is only 127kb in size. Also, the boot partition apparently can not be mounted:

![image](https://user-images.githubusercontent.com/422437/75925492-19769400-5e69-11ea-97be-4983faf6c108.png)

`df -h` shows that the application partition is very small:

![image](https://user-images.githubusercontent.com/422437/75925561-2b583700-5e69-11ea-9833-df4d477bb439.png)

### Expected behavior

The application partition should have at least a few hundred megabytes with an unmodified `fwup.conf`.

**Comment by fhunleth** [comment]
I can't reproduce right now, but I'm curious if you have any other hints. Like is this repeatable? I don't know how it would affect it, but are there any details about your system that might be different. Also, could you run `dmesg` and post the results?

**Comment by krgn** [comment]
Thanks for you reply! Here is my fork:

https://github.com/krgn/custom_x86_64

I have barely changed anything from the defaults, except including the docker cli/daemon. Since I am running NixOS as my dev machine, I am using a custom ubuntu focal docker image for building.  

What is also strange is that dhcp is not present in the image either, so its difficult to get the full `dmesg` output dumped out (other than as screenshots). 

I also realized that I had based my fork off of the `v1.9.2` tag, and just updated from master. I also added `dhcp` to the build, so I can ssh into qemu. I'll report back my findings. :)

**Comment by krgn** [comment]
Same issue with current HEAD. 

![image](https://user-images.githubusercontent.com/422437/75960129-ced43680-5ebf-11ea-86ed-97cbaf40fea1.png)
![image](https://user-images.githubusercontent.com/422437/75960154-d5fb4480-5ebf-11ea-8643-0de1074ea4c1.png)
![image](https://user-images.githubusercontent.com/422437/75960165-db588f00-5ebf-11ea-8acf-54024f27c1d3.png)

`dhclient` also does not work:

![image](https://user-images.githubusercontent.com/422437/75960225-f7f4c700-5ebf-11ea-87df-87ff147ebf6a.png)

![image](https://user-images.githubusercontent.com/422437/75960254-02af5c00-5ec0-11ea-83d8-305326002eba.png)

![image](https://user-images.githubusercontent.com/422437/75960308-1b1f7680-5ec0-11ea-9ff7-023fddadfae5.png)

**Comment by fhunleth** [comment]
Sorry for the super long time to address this. I have `fwup` expand the data partition so it "just works". Fixed in #172.
